autoPyTorch/pipeline/components/training/data_loader/base_data_loader.py

```diff
@@ -31,9 +31,11 @@ class BaseDataLoaderComponent(autoPyTorchTrainingComponent):
 
     """
 
-    def __init__(self, batch_size: int = 64) -> None:
+    def __init__(self, batch_size: int = 64,
+                 random_state: Optional[np.random.RandomState] = None) -> None:
         super().__init__()
         self.batch_size = batch_size
+        self.random_state = random_state
```

Comment:
should we not send this random_state to the super class? So it is handled similar to how it is handled in the base component